src/unity/Assets/Crest/Shaders/Ocean.shader

```diff
@@ -379,7 +381,18 @@ Shader "Ocean/Ocean"
 					half3 col = OceanEmission(view, n_pixel, lightDir, i.grabPos, pixelZ, uvDepth, sceneZ, sceneZ01, bubbleCol, _Normals, _CameraDepthTexture, underwater, scatterCol);
 
 					// Light that reflects off water surface
+					#if _UNDERWATER_ON
+					if (underwater)
```

Comment:
Minor comment - you could avoid the duplicate calls to reflection wrapping the if/else:

#if ...
if(underwater)
{ ...}
else
#endif
{
  reflection sky ()
}

I personally find this easier to follow but it might just be me, ill leave it up to you.